wagtail/core/views.py

```diff
@@ -1,6 +1,8 @@
+from django.conf import settings
 from django.http import Http404, HttpResponse
 from django.shortcuts import get_object_or_404, redirect
 from django.urls import reverse
+from django.utils.http import is_safe_url
```

Comment:
Please note: `django.utils.http.is_safe_url()` is deprecated in favor of `url_has_allowed_host_and_scheme()`

Reply:
👍 I'll make the change.